apps/etterlatte-behandling-kafka/src/main/kotlin/no/nav/etterlatte/brukerdialog/soeknad/client/BehandlingClient.kt

```diff
@@ -40,6 +40,7 @@ class FeiletVedOpprettBehandling(
 )
 class BehandlingClient(
     private val sakOgBehandlingApp: HttpClient,
+    private val behandlingAppForventSukess: HttpClient,
```

Comment:
Dropp to klienter her og håndter det ene endepunktet (opprettBehandling) som har behov for en catch og rethrow ved forventetSuksess=true